plugins/maven-dependency-resolver/build.gradle

```diff
@@ -3,13 +3,55 @@ import org.robolectric.gradle.RoboJavaModulePlugin
 
 apply plugin: RoboJavaModulePlugin
 apply plugin: DeployedRoboJavaModulePlugin
+apply plugin: 'kotlin'
+apply plugin: "com.diffplug.spotless"
+
+spotless {
+    kotlin {
+        target '**/*.kt'
+        ktfmt('0.42').googleStyle()
+    }
+}
+
+tasks.withType(GenerateModuleMetadata) {
```

Comment:
```suggestion
tasks.withType(GenerateModuleMetadata).configureEach {
```

Reply:
I remove it at gradle.properties to avoid add stdlib info to generated library metadata. So I must to add it for testing.